src/engine/strat_engine/engine.rs

```diff
@@ -171,6 +250,10 @@ impl Engine for StratEngine {
         get_mut_pool!(self; uuid)
     }
 
+    fn configure_simulator(&mut self, _denominator: u32) -> EngineResult<()> {
```

Comment:
I think this move of ```configure_simulator``` belongs in a completely different PR, or is a rebasing mistake.

Reply:
Rebase error, addressed with another rebase and merge conflict